mkt/ratings/views.py

```diff
@@ -67,17 +68,21 @@ def filter_queryset(self, queryset):
         Custom filter method allowing us to filter on app slug/pk and user pk
         (or the special user value "mine"). A full FilterSet is overkill here.
         """
-        filters = {}
+        filters = Q()
         app = self.request.GET.get('app')
         user = self.request.GET.get('user')
+        lang = self.request.GET.get('lang')
+        match_lang = self.request.GET.get('match_lang')
         if app:
             self.app = self.get_app(app)
-            filters['addon'] = self.app
+            filters &= Q(addon=self.app)
         if user:
-            filters['user'] = self.get_user(user)
+            filters &= Q(user=self.get_user(user))
+        elif lang and match_lang == '1':
+            filters &= Q(user__lang=lang)
```

Comment:
AFAIK `user.lang` is not indexed so this will be costly... I think it'd be better to run the migration to populate `lang` on the `Review` model first, and then use that directly in this PR.